docs/manual/manual_stock_led.rst

```diff
@@ -0,0 +1,314 @@
+purchasing more LED add-on boards, please contact us at info@vertiq.co. Please refer to your module's family page to see if it belongs to Gen 1 or Gen 2.
+
+Hardware Configuration
+=========================
+If you are using Vertiq's Pro Kit, simply plug the LED connector into the LED board, and you can move on to *Controlling Your LEDs*. Otherwise continue reading.
+
+Vertiq's Gen 2 modules all feature solder pads that allow you to easily connect with external power sources and other peripherals. The LED peripheral board connects 
+with your module's GPIO2 and GPIO3 connections where GPIO2 controls the RGB LED and GPIO3 controls the white LEDs. Depending on the size of your module, 
+you will see the GPIO values printed as either IOX or GPIOX where X is the GPIO value.
+
+.. list-table:: LED Connection Examples
+   :class: borderless
+   :align: center
+
+   * - .. figure:: ../_static/manual_images/leds/40_led_connections.png
+            :alt: 40-06 LED Connections
+            :align: center
+            :height: 350
+
+            Vertiq 40-06 LED Connections
+     - .. figure:: ../_static/manual_images/leds/81_led_connections.png
+            :alt: Vertiq 81-08 Gen 2 LED Connections
+            :align: center
+            :height: 350
+
+            Vertiq 81-08 Gen 2 LED Connections
+
+The LED board's power connections should be tied to your module's V+ and V- ports ensuring that GND attaches to V-, and V+ connects to V+. The LED board has an operating range of 
+3-14S (11.1-52V nominal). Operating the LED board above this range can result in serious damage to the board. Operating the board below this range will simply prevent it from 
+operating as expected.
+
+The LED board uses a 4-pin JST-ZE connector in order to interface with your module. The following image displays the LED board's pinout.
+
+.. image:: ../_static/manual_images/leds/led_board_connections.png
+
+Please note that the connector's clip is at the top of this image. Take care to ensure that the module's RGB LED pad connects with the LED board's RGB LED pin, 
+and the module's white LED pad attaches to the LED board's white LED pin. Failure to do so can result in damage to your LED board as well as your module.
+
+Controlling Your LEDs
+========================
+.. warning::
+    Do not plug in your module's LED board while the module is already powered up. “Hot plugging” your module this way can cause damage to the LED board as well 
+    as the module itself. Always ensure that the LED board is fully connected with the module before powering it on.
+
+.. _default_pattern:
+
+Default Pattern
+******************
+
+By default, Vertiq modules control the RGB LED to display a static blue color, and the white LEDs to be shut off. On startup, the RGB LED should be illuminated.
+
+.. image:: ../_static/manual_images/leds/vertiblue_default_on.png
+    :width: 60%
+
```

Comment:
Its a little funky that the images are offset to the left, and the videos are centered, its super obvious right here and makes me feel weird:
![image](https://github.com/user-attachments/assets/415f2c3b-4468-41b2-8dd2-52e92c90893d)


Any way to make them have the same formatting? Either centering the images or left aligning the video?